hveto/__main__.py

```diff
@@ -924,10 +927,11 @@ def main(args=None):
                 timeout=4,
                 extra_commands=["request_disk=1G"],
                 gps=start)
+            do_submit = True if args.no_submit else False
```

Comment:
Should this be the other way around? To me this reads 'do the submit if the 'no submit' option was selected', which seems upside down.

Reply:
Also, if this is just toggling the boolean option, just use the boolean option:

```suggestion
            do_submit = args.no_submit
```

or

```suggestion
            do_submit = not args.no_submit
```